Dockerfile

```diff
@@ -29,7 +29,8 @@ ENV IMMUDB_HOME="/usr/share/immudb" \
     IMMUDB_DEVMODE="true" \
     IMMUDB_MAINTENANCE="false" \
     IMMUDB_ADMIN_PASSWORD="immudb" \
-    IMMUADMIN_TOKENFILE="/var/lib/immudb/admin_token"
+    IMMUADMIN_TOKENFILE="/var/lib/immudb/admin_token" \
+	IMMUDB_PGSQL_SERVER="false"
```

Comment:
Let's keep the IMMUDB_PGSQL_SERVER enabled by default.

It's always possible to disable it by running with `IMMUDB_PGSQL_SERVER` env var set to `false`.

Reply:
@byo done